src/MoBi.Presentation/DTO/ModuleContentDTO.cs

```diff
@@ -10,7 +10,19 @@ public class ModuleContentDTO : ObjectBaseDTO, IWithProhibitedNames
       public bool WithPassiveTransport { get; set; }
       public bool WithMolecule { get; set; }
       public bool WithObserver { get; set; }
-      public virtual bool WithInitialConditions { get; set; }
+
+      private bool _withInitialConditions;
+
+      public virtual bool WithInitialConditions
+      {
+         get => _withInitialConditions;
+         set
+         {
+            _withInitialConditions = value;
+            OnPropertyChanged(nameof(WithInitialConditions));
```

Comment:
that's not required if you use the method that is Defied SetProperty(ref, value) sthg like this